rules/react.js

```diff
@@ -494,7 +499,7 @@ module.exports = {
 
     // Enforce that props are read-only
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-read-only-props.md
-    'react/prefer-read-only-props': 'off',
+    'react/prefer-read-only-props': 'error',
```

Comment:
This might lead to a lot of changes but I think is most likely simple fixes. Not sure if it would be too verbose though. 🤔

Reply:
I think I'll enable them one-by-one to have separated commits